packages/components/src/FormField/FormField.css

```diff
@@ -53,7 +53,9 @@
 }
 
 .wrapper:focus-within {
-  --field--border-color: var(--color-focus);
+  position: relative;
+  z-index: var(--field--base-elevation);
```

Comment:
Referencing the CSS approach used by [ANT Financial](https://ant.design/components/input/#components-input-demo-group) to handle the same "show focus state even when subsequent inputs should overlap" situation

Reply:
Cool